server/publicapi/packages/service.py

```diff
@@ -9,13 +9,13 @@
 # at https://www.sourcefabric.org/superdesk/license
 
 import logging
-from superdesk.services import BaseService
+from publicapi.items.service import ItemsService
 
 
 logger = logging.getLogger(__name__)
 
 
-class PackagesService(BaseService):
+class PackagesService(ItemsService):
```

Comment:
Currently the whole item functionality is needed in packages so it makes sense for now. If and when item and package functionality diverge creating a base class for them will be trivial. Currently as Petr says I think it's premature optimization :)